synapse/axon.py

```diff
@@ -1015,6 +1019,74 @@ async def jsonlines(self, sha256):
                 raise s_exc.BadJsonText(mesg=f'Bad json line encountered while processing {sha256}, ({e})',
                                         sha256=sha256) from None
 
+    async def postfiles(self, fields, url, params=None, headers=None, method='POST', ssl=True, timeout=None):
```

Comment:
Could the fields param become an optional param to the regular request API?

Reply:
I think it fits in pretty well over there.